docs/_config.yml

```diff
@@ -15,6 +15,8 @@ collections:
     output: false
   steps:
     output: false
+  deploying:
```

Comment:
if we use the header "deployment", we should change this as well.

Reply:
Fixed in f628553!